<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_emailSettings</class>
 <widget class="QWidget" name="form_emailSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки электронной почты</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>SMTP-адрес:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Порт:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editSMTPAddress">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Например, smtp.example.com</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editSMTPPort">
      <property name="inputMask">
       <string>00000</string>
      </property>
      <property name="placeholderText">
       <string>Например, 666</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editLogin"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="buttonErase">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="buttonSave">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonErase</sender>
   <signal>clicked()</signal>
   <receiver>editLogin</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonErase</sender>
   <signal>clicked()</signal>
   <receiver>editPassword</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonErase</sender>
   <signal>clicked()</signal>
   <receiver>editSMTPAddress</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonErase</sender>
   <signal>clicked()</signal>
   <receiver>editSMTPPort</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>form_emailSettings</receiver>
   <slot>buttonSave_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonSave_clicked()</slot>
  <slot>form_destroyed()</slot>
 </slots>
</ui>
